content/docs/get-started/management.md

```diff
@@ -0,0 +1,218 @@
+scikit-learn and train a simple classifier. But instead of pickling the model,
+we save it with MLEM (check out the full list of supported
+[ML frameworks](/doc/object-reference/model)).
+
+Once you saved the model with MLEM, you can load it to use in a Python runtime,
+or generate predictions for a dataset stored on the disk from the command line.
+
+## Loading a model
+
+Now, we can use MLEM to load the model and calculate some metrics:
+
+```py
+# predict.py
+from mlem.api import load
+
+model = load("models/rf")  # RandomForestClassifier
+features = [
+    "sepal length (cm)",
+    "sepal width (cm)",
+    "petal length (cm)",
+    "petal width (cm)",
+]
+x = pd.DataFrame([[0, 1, 2, 3]], columns=features)
+y_pred = model.predict_proba(x)
+
+print(y_pred)
+```
+
+Now, let's run the script
+
+```cli
+$ python predict.py
+[[0.47 0.24 0.29]]
+```
+
+We see that the prediction was successfully printed into stdout.
+
+## Batch scoring in CLI
+
+In a Batch scoring scenario you often want to apply your model to a dataset from
+a command line to get instant feedback about how your model behaves. Let's see
+how MLEM can help by creating an example file and running `mlem apply`:
+
+```cli
+$ echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3" > new_data.csv
+
+$ mlem apply models/rf new_data.csv \
+    --method predict_proba \
+    --import --it "pandas[csv]"
+⏳️ Importing object from new_data.csv
+⏳️ Loading model from models/rf.mlem
+🍏 Applying `predict_proba` method...
+[[0.47 0.24 0.29]]
+```
+
+<details>
+
+### Learn more about `--method`, `--import` and `--it` options used
+
+- The `--method`/`-m` flag tells MLEM to invoke the `predict_proba` method and
+  return the class probabilities, instead of the default `predict`.
+- The `--import`/`-i` flag tells MLEM to import the data on the fly.
+- The `--import-type` / `--it` flag, helps MLEM understand the data format.
+  Here, it's `pandas[csv]` a csv file that should be read with Pandas. For that
+  to work, your data should be in a format that is supported by
+  [MLEM import](/doc/user-guide/importing). You can learn more about specifying
+  these arguments on `mlem apply` page.
+
+Alternatively, you could save the [data with MLEM](/doc/user-guide/data) to use
+`mlem apply` on it.
+
+</details>
+
+## Model Codification
+
+### Why do it the MLEM way
+
+Saving models to files or loading them back into python objects may seem like a
+deceptively simple task at first. For example, `pickle` and `torch` libraries
+can serialize/deserialize model objects to/from files, but we will see that MLEM
+adds some "special sauce" in the form of metadata files that will help us a lot
+down the line in the heavier operations like packaging and serving of the models
+in various ways. MLEM allows us to automate a lot of the pain points we would
+hit later on in our ML workflow by codifying metadata about your models (and
+other objects) and intelligently using it later on.
```

Comment:
BTW, great info but too verbose for the Get Started. We fight for every word in this section. Let's move it to a guide somewhere?